compose.go

```diff
@@ -123,6 +123,7 @@ type Generator struct {
 	RemoveVolumes           bool
 	NoCreateRootTarget      bool
 	AutoFormat              bool
+	Formatter 				string
```

Comment:
Please format the file. You can either set this up in your editor, or use `go fmt`:

```
go fmt ./...
```

Reply:
:+1: